graph/traverse/traverse.go

```diff
@@ -27,22 +27,34 @@ type Graph interface {
 
 // BreadthFirst implements stateful breadth-first graph traversal.
 type BreadthFirst struct {
-	EdgeFilter func(graph.Edge) bool
-	Visit      func(u, v graph.Node)
-	queue      linear.NodeQueue
-	visited    set.Int64s
+	// Visit is called on all nodes on their first visit.
+	Visit func(graph.Node)
+
+	// Traverse is called on all edges that may be traversed
+	// during the walk. This includes edges that would hop to
+	// an already visited node.
+	//
+	// The value returned by EdgeFulter determines whether an
```

Comment:
Typo. `EdgeFulter` should be `EdgeFilter` and now `Traverse`.
```suggestion
	// The value returned by Traverse determines whether an
```

Reply:
Thanks. Fixed.